components/sup-client/src/lib.rs

```diff
@@ -91,7 +91,10 @@ impl error::Error for SrvClientError {
 impl fmt::Display for SrvClientError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let content = match *self {
-            SrvClientError::ConnectionClosed => "Connection closed".to_string(),
+            SrvClientError::ConnectionClosed =>
+                "The connection was unexpectedly closed.\n\n\
+                This may be because the given Supervisor is in the middle of an orderly shutdown,\n\
+                and is no longer processing command requests.".to_string(),
```

Comment:
YAY! for this clarity. I'm pretty sure I lost a good chunk of time seeing `Connection closed` and thinking "OMG Connection Closed?!" to later realize "oh its ok...This may be because the given Supervisor is in the middle of an orderly shutdown"